ui/src/components/SubPlayPage.tsx

```diff
@@ -1,27 +1,58 @@
+import { METHODS } from "http";
 import React, { useState } from "react";
 import DrawingCanvas from "./DrawingCanvas";
 
-const SubPlayPage = ({number_truth = "0"}) => {   
+const SubPlayPage = ({number_truth = "0"}) => {
+    
+    const url = "http://192.168.1.65:8000/predict"
+    const [imageData, setImageData] = useState<Blob | null>(null)
```

Comment:
i saw the interface Blob but do not know where you define in this file.

Reply:
Blob is a build-in javascript datatype.